modal/_container_entrypoint.py

```diff
@@ -416,8 +416,8 @@ def run_input_sync(io_context: IOContext) -> None:
                 )
         reset_context()
 
-    if input_concurrency > 1:
-        with DaemonizedThreadPool(max_threads=input_concurrency) as thread_pool:
+    if target_input_concurrency > 1:
+        with DaemonizedThreadPool(max_threads=target_input_concurrency) as thread_pool:
```

Comment:
Would this limit you to the initial concurrency? Maybe we should remove the `max_threads`?